python-challenges/email-scheduler/devashmit-day19/scheduler.py

```
@@ -0,0 +1,46 @@
+"""Email Automation Scheduler — Day 19 Python | Author: devashmit"""
+import argparse,csv,json,logging,os,smtplib
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+from pathlib import Path
+
+logging.basicConfig(filename="scheduler.log",level=logging.INFO,format="%(asctime)s %(levelname)s %(message)s")
+SMTP_HOST=os.getenv("SMTP_HOST","smtp.gmail.com");SMTP_PORT=int(os.getenv("SMTP_PORT","587"))
```

Ruff (0.15.17)

[error] Multiple statements on one line (semicolon)

(E702)